### PR TITLE
added 33 utility docs

### PR DESCRIPTION
**Pull request by EcaleD**
closes: #ISSUE_Number

---

## Change logs

> Describe your change clearly, including what problem is being solved or what document is being added or updated.

added 33 utility docs; fixed some link issues

## Contributor's checklist

Here are some reminders before you submit your pull request:

* Make sure that your Pull Request has a clear title and commit message. You can take the [Git commit template](https://github.com/cloudberrydb/cloudberrydb/blob/main/.gitmessage) as a reference.
* Sign the Contributor License Agreement as prompted for your first-time contribution (*One-time setup*).
* Learn the [code contribution](https://cloudberrydb.org/contribute/code) and [doc contribution](https://cloudberrydb.org/contribute/doc) guides for better collaboration.
* Make sure that your changes deployment preview is successful.
* List your communications in the [GitHub Issues](https://github.com/cloudberrydb/cloudberrydb-site/issues) or [Discussions](https://github.com/orgs/cloudberrydb/discussions) (if has or needed).
* Feel free to ask for the @cloudberrydb/doc team or other people to help review and approve.

### CODE REVIEW

File: docs/sys-utilities/gpactivatestandby.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+Activate the standby coordinator host and make it the active coordinator instance for a Cloudberry Database system (run from backup coordinator host you are activating):
+
+```shell
+gpactivatestandby -d /gpdata
+```
+
+## See also
+
+[gpinitsystem](/docs/db-utilities/db-util-gpinitsystem.md), [gpinitstandby](/docs/db-utilities/db-util-gpinitstandby.md)
```
Comment (tuhaihe) [inline docs/sys-utilities/gpactivatestandby.md:96]: the dir name has changed from `db-utilities` to `sys-utilities`.

Comment (EcaleD) [reply docs/sys-utilities/gpactivatestandby.md]: Thanks for the comment. Updated the value in links.

**Review by tuhaihe — APPROVED** [review]
LGTM.

**Comment by tuhaihe** [comment]
> In the `sys-utilities` directory, some files are named as the command-line tool name (like `analyzedb.md` and `clusterdb.md`) while some are named as `db-util-xx.md`. Maybe we can use a unified naming rule for all the file names.

I can help unify the file name formats in the following days. How about change "db-util-xx.md" to "xx.md"? Those files are all under the `db-util` dir. So I think there is no need to keep the `db-util-` prefix for the file name. @TomShawn

**Pull request merged**